src/fe/implicit/_Obj.js

```diff
@@ -272,3 +272,30 @@ export const entries = o => {
 
   return list;
 };
+
+/**
+ * Returns the value at a path if value exists
+ * @param {Object} object
+ * @param {string} path
+ *
+ * @returns {*}
+ */
+export const getSafely = (object, path) => {
+  const points = path.split('.');
+  let anchor = object;
+
+  for (let i = 0; i < points.length; i++) {
+    try {
+      const item = anchor[points[i]];
+      if (!item) {
```

Comment:
Add an explicit `typeof item === 'undefined'` check here. This will break if we try to access the length of an empty string.

Reply:
`undefined` isn't the only condition where we should stop. `null`, `bool` - all of those need to be considered too. We'll need to think of a better way to handle this.